From 67457005e2d42603e834a923a9151ce8da50addd Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 8 Oct 2019 12:04:26 -0700
Subject: [PATCH] py3cairo is only pycairo for Clear

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0d90514..7c67a1f 100755
--- a/setup.py
+++ b/setup.py
@@ -1941,7 +1941,7 @@ if client_ENABLED and gtk3_ENABLED:
     if OSX or is_Ubuntu() or is_Debian():
         pycairo = "py3cairo"
     else:
-        pycairo = "pycairo"
+        pycairo = "py3cairo"
     cython_add(Extension("xpra.client.gtk3.cairo_workaround",
                 ["xpra/client/gtk3/cairo_workaround.pyx"],
                 **pkgconfig(pycairo)
-- 
2.23.0

