--- ./xpra/client/gtk3/cairo_workaround.pyx~	2019-08-02 10:30:08.940858748 -0700
+++ ./xpra/client/gtk3/cairo_workaround.pyx	2019-08-02 10:31:38.259857897 -0700
@@ -63,7 +63,7 @@
     void cairo_surface_flush (cairo_surface_t *surface)
     void cairo_surface_mark_dirty (cairo_surface_t *surface)
 
-cdef extern from "pycairo/py3cairo.h":
+cdef extern from "py3cairo.h":
     ctypedef struct Pycairo_CAPI_t:
         pass
     ctypedef struct PycairoSurface:
